eva/expression/aggregation_expression.py

```diff
@@ -55,9 +55,14 @@ def evaluate(self, *args, **kwargs):
         elif self.etype == ExpressionType.AGGREGATION_MAX:
```

Comment:
We miss an `else` condition to raise an error. Right now, we silently ignore it and return the origin batch.